zipline/lib/adjusted_array.py

```diff
@@ -227,6 +227,7 @@ def traverse(self,
             offset,
             window_length,
             perspective_offset,
+            rounding_places=None,
```

Comment:
Correct, for instance in pipeline, if there's a price adjustment that causes prices to go out to more than 3 decimal places, I don't believe any rounding currently occurs.